gradle.properties

```diff
@@ -1,3 +1,3 @@
 
-version = 1.0.0
+version = 4.0.0
```

Comment:
there is a version 3.5.0 in the s3 maven2 bucket, so not sure why this was on 1.0.0